gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -363,6 +363,8 @@ MM_VerboseHandlerOutput::outputInitializedStanza(MM_EnvironmentBase *env, MM_Ver
 
 	buffer->formatAndOutput(env, 1, "<system>");
 	buffer->formatAndOutput(env, 2, "<attribute name=\"physicalMemory\" value=\"%llu\" />", omrsysinfo_get_physical_memory());
+	buffer->formatAndOutput(env, 2, "<attribute name=\"using container\" value=\"%s\" />", OMR_CGROUP_SUBSYSTEM_MEMORY == omrsysinfo_cgroup_are_subsystems_enabled(OMR_CGROUP_SUBSYSTEM_MEMORY)?"true":"false");
```

Comment:
sorry misread that you broke it into 2 lines...

Reply:
to be precise there is also case 50% if total memory is less than 1G

Reply:
So I guess we can replace both by one single line showing whether we or using 75% or 25% here?

Reply:
> to be precise there is also case 50% if total memory is less than 1G

This is just a case within container path, right?

I said 75% vs 25%, but effectively I want to report whether  we chose 'container limit' path/case or not. We don't have to report details about all sub-cases within those 2 main cases (for example, why we chose 50%, but not 75% while in 'container limit' case).

I think that one line would be helpful, but not too much going into details for user to start think what other flags mean and risking too much for future if we change something about these flags.

Reply:
I like idea to report "container limit selected". However I don't think that checks should be done in Verbose log code in this case. Direct checks in Port are good for general environment data. Decisions data should be set at the moment of selection in GC code, like `extensions->containerLimitSelected` set to true in `MM_GCExtensions::computeDefaultMaxHeapForJava()` and used in Verbose code.

Reply:
... or alternatively just report here in one line "run in container with memlimit set to <value>" (not sure "container" is generalized word enough. Also might be it is a place to report about memlimit.

Reply:
I'm ok with cashing the result of 'container limit set' in GC Ext and using that in VGC.

But I don't think that we need to report what that limit is. That limit already affected omrsysinfo_get_addressable_physical_memory() value (and our internal usablePhysicalMemory cached value) which we should report as a separate line regardless if container limit is set or not (or if it's the same or different value than  omrsysinfo_get_physical_memory() value). I think this is just a slightly more generic way of reporting in case omrsysinfo_get_addressable_physical_memory() is affected by some other things than the container limit. 

If we are sure that the containter limit is always (the only one) reason that will make omrsysinfo_get_addressable_physical_memory() be different/lower than omrsysinfo_get_physical_memory(), then I'm ok reporting it as a container limit value.

Reply:
Again, the point of the flag is just to clarify why reported maxHeapSize looks like 75/50% (because container limit was set) vs 25% (because container limit was not set) of addressablePhysical memory.

Reply:
ok, I agree to keep it more general. Should we have formal "selection reason" variable to be reported (there are two at the moment, "general" and "container") ?